applications/chronicle.md

```diff
@@ -0,0 +1,204 @@
+    Zaisan.io, working under "cc32d9" pseudonym.
+
+- Daniel Liven, managing the legal and financial aspects.
+
+### Legal Structure
+- **Registered Legal Entity:** Zaisan BV
+- **Registered Address:** Keizersgracht 391A, 1016 EJ, Amsterdam, The Netherlands
+
+### Team Experience
+
+Zaisan has been around in EOSIO ecosystem for several years, formally
+known as Europechain BV. The company is founded by several European
+block producers, and is focusing on software projects and business
+consultancy.
+
+The company has previously taken the task of writing the EOSIO Core+
+Blue Paper on request of ENF. 
+
+### Team Org Repos
+
+- https://github.com/Europechain
+- https://github.com/eos-amsterdam-rnd
+
+### Team Member Repos
+
+- https://github.com/cc32d9
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/dliven/
+
+## Development Status
+
+- [Initial proposal and crowdfunding proceedings](https://github.com/cc32d9/eos-work-proposals/tree/master/001_EOS_Chronicle)
+- [Main project repository](https://github.com/EOSChronicleProject/eos-chronicle)
+- [Chronicle tutorial](https://github.com/EOSChronicleProject/chronicle-tutorial)
+- [JavaScript NPM for Chronicle consumers](https://github.com/EOSChronicleProject/chronicle-consumer-npm)
+- Blog publications: [fundraising](https://cc32d9.medium.com/fundraising-for-chronicle-and-history-indexer-26e0a06c2d1d), [overview of history solutions](https://cc32d9.medium.com/history-and-notifications-in-eosio-blockchain-8255194af93)
+
+## Development Roadmap
+
+### Overview
+
+- **Total Estimated Duration:**  4 months
+- **Full-Time Equivalent (FTE):**  0.25
+- **Total Costs:** USD 20,000
+
+### Milestone 1 — Chronicle software update
+
+- **Estimated duration:** 2 months
+- **FTE:**  0.25
+- **Costs:** 8,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 0a. | License | Apache 2.0 |
+| 0b. | Documentation | Updated Chronicle README and the Tutorial to reflect all the recent changes |
+| 0c. | Testing Guide | Tests scripts and guides will be part of documentation |
+| 1. | Update dependencies | Update Chronicle to work with latest releases of dependency libraries (Boost.org and EOSIO/Mandel libraries).|  
+| 2. | Ubuntu packages | Crate package build scripts; publish the packages for public download |   
+| 3. | JavaScript NPM | Redesign the asynchronous mode to avoid memory overflow |  
+| 4. | Third party contributions | Integrate, test and document third-party contributions, such as Docker and Kubernetes images |
+
+
+### Milestone 2 — Research
+
+- **Estimated duration:** 2 month
+- **FTE:**  0.25
+- **Costs:** 8,000 USD
+
+| Number | Deliverable | Specification |
+| -----: | ----------- | ------------- |
+| 1 | Alternative DB backend | Current Chronicle uses EOSIO Chainbase for its data. An alternative, more lightweight and less demanding, backend needs to be researched, and possibly taken for the next release. |
```

Comment:
What is the deliverable on this milestone? Is this just a proposed DB backend or a fully implemented alternative?